tests/very_simple.to

```diff
@@ -0,0 +1,3 @@
+int n = 3;
+
+bool b = null;
```

Comment:
I know this is just a parser test, but is this allowed?

Reply:
In the current specification of Teko, all types are reference types and therefore nullable... I suppose that could change.

I was thinking about a Maybe type wrapper as in Haskell, perhaps expanded a little bit to do some error propagation similarly to Go. If I had that I wouldn't imagine there's a need for everything to be nullable.

Reply:
Yeah, I can understand object types being nullable, but allowing null booleans seems like asking for trouble. Anyway, not really relevant to this PR.